sunbird_service_api_test/src/test/java/org/sunbird/integration/test/malformedRequest/MalformedRequestTest.java

```diff
@@ -0,0 +1,121 @@
+						"/api/user/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createOrg",
+						"/api/org/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createCourse",
+						"/api/course/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createPage",
+						"/api/page/v1/create" },
+				new Object[] { emptyPayLoad,
+						Constant.RESPONSE_TEMPLATE_LOCATION + "content_type_mandatory_response.json", "createNote",
+						"/api/note/v1/create" } };
+	}
+
+	/**
+	 * Test for create request without content-type header.
+	 *
+	 * @param requestJson
+	 * @param responseJson
+	 * @param testName
+	 */
+	@Test(dataProvider = "createRequestDataProvider", dependsOnMethods = { "getAdminAuthToken", })
+	@CitrusParameters({ "requestJson", "responseJson", "testName", "url" })
+	@CitrusTest
+	public void createWithoutContentType(String requestJson, String responseJson, String testName, String url) {
+		getTestCase().setName(testName);
+		System.out.println("request: == " + requestJson);
+		http().client(restTestClient).send().post(url)
+				.header(Constant.AUTHORIZATION, Constant.BEARER + initGlobalValues.getApiKey())
+				.header(Constant.X_AUTHENTICATED_USER_TOKEN, admin_token).payload(requestJson);
+
+		http().client(restTestClient).receive().response(HttpStatus.BAD_REQUEST)
+				.payload(new ClassPathResource(responseJson));
+
+	}
+
+	/**
+	 * Test for create request with invalid(json-ld) content-type header.
+	 *
+	 * @param requestJson
+	 * @param responseJson
+	 * @param testName
+	 */
+	@Test(dataProvider = "createRequestDataProvider", dependsOnMethods = { "getAdminAuthToken", })
+	@CitrusParameters({ "requestJson", "responseJson", "testName", "url" })
+	@CitrusTest
+	public void createWithInvalidContentType(String requestJson, String responseJson, String testName, String url) {
+		getTestCase().setName(testName);
+		http().client(restTestClient).send().post(url).contentType(Constant.CONTENT_TYPE_APPLICATION_JSON_LD)
+				.header(Constant.AUTHORIZATION, Constant.BEARER + initGlobalValues.getApiKey())
+				.header(Constant.X_AUTHENTICATED_USER_TOKEN, admin_token).payload(requestJson);
+
+		http().client(restTestClient).receive().response(HttpStatus.BAD_REQUEST)
+				.payload(new ClassPathResource(responseJson));
+
+	}
+
+	@Test()
+	@CitrusTest
+	/**
+	 * Key cloak admin token generation is required , because on sunbird dev server
+	 * after creating user , user have to login first then only his/her account will
+	 * be active. so we need to disable that option for created user only. That
+	 * option can be disable using keycloak admin auth token. So this method will
+	 * generate auth token and that token will be used in
+	 * **updateUserRequiredLoginActionTest** method.
+	 */
+	public void getAdminAuthToken() {
```

Comment:
Can we put it into common place , to reuse it.

Reply:
we can decide later whether can be moved into common baseTest for all testcases